cloud_functions/shared/cloud_utils.py

```
@@ -5,26 +5,28 @@
             request={
                 "name": f"projects/{environ['PROJECT_ID']}/secrets/{name}/versions/latest",
             }
         )
-        logger.debug(f"Retrieved secret {name}")
+        partial_log_structured(message=f"Retrieved secret {name}")
         secret_payload = response.payload.data.decode("UTF-8")
         return json.loads(secret_payload)
     except Exception as e:
-        logger.error(f"Failed to retrieve or decode secret {name}: {e}")
+        partial_log_structured(
+            message=f"Failed to retrieve or decode secret {name}: {e}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)